pipelines/utilities/longhorn_helm_chart.sh

```
@@ -1,15 +1,25 @@
+
+set -x
 
+source pipelines/utilities/longhorn_status.sh
 
 get_longhorn_chart(){
   CHART_VERSION="${1:-$LONGHORN_REPO_BRANCH}"
+
+  # create and clean tmpdir
+  TMPDIR="/tmp/longhorn"
+  mkdir -p ${TMPDIR}
+  rm -rf "${TMPDIR}/"
```

Shellcheck (0.10.0)

[warning] Use "${var:?}" to ensure this never expands to / .

(SC2115)